packages/react-magma-dom/src/components/Dropdown/DropdownExpandableMenuPanel.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+import { AccordionPanel, AccordionPanelProps } from '../Accordion';
+import { DropdownExpandableMenuItemContext } from './DropdownExpandableMenuItem';
+
+export interface DropdownExpandableMenuPanelProps extends AccordionPanelProps {
+  testId?: string;
+  isExpandablePanel?: boolean;
+}
+
+const StyledAccordionPanel = styled(AccordionPanel)<{
+  hasIcon?: boolean;
+  isExpandablePanel?: boolean;
+}>`
+  padding: 0;
+`;
+
+export const DropdownExpandableMenuPanel = React.forwardRef<
+  HTMLDivElement,
+  DropdownExpandableMenuPanelProps
+>(props => {
+  const { children, isExpandablePanel, testId } = props;
+
+  const context = React.useContext(DropdownExpandableMenuItemContext);
+
+  // const theme = React.useContext(ThemeContext);
+
+  // const ref = React.useRef();
+
+  // const i18n = React.useContext(I18nContext);
+
+  return (
+    <StyledAccordionPanel
+      hasIcon={context.hasIcon}
+      isExpandablePanel={isExpandablePanel}
```

Comment:
Should we be using `isExpandablePanel` from the `DropdownExpandableMenuItemContext` or from props?

Reply:
Refactored and no longer a prop in this component.